lib/elm-test.js

```diff
@@ -165,12 +169,12 @@ function main() {
   program
     .command('init')
     .description('Install elm-explorations/test and create tests/Example.elm')
-    .action(() => {
+    .action(async () => {
       const options = program.opts();
       const pathToElmBinary = getPathToElmBinary(options.compiler);
       const project = getProject('init');
       try {
-        Install.install(project, pathToElmBinary, 'elm-explorations/test');
+        await Install.install(project, pathToElmBinary, 'elm-explorations/test');
```

Comment:
Does `Install.install` return a promise now? See the other comment about `async`/`await`. I’d extract most stuff here to a helper function I think, where I can use `async`/`await`, and then call it here with `.then` and `.catch` like the other command.s

Reply:
Good catch, thanks! I previously implemented this hacking of elm-explorations/test inside Install.install before realizing that's the wrong place, and didn't fully revert the changes to Install.install.

Reverted in ae25add